src/net_processing.cpp

```diff
@@ -757,7 +774,7 @@ class PeerManagerImpl final : public PeerManager
     /** Update pindexLastCommonBlock and add not-in-flight missing successors to vBlocks, until it has
      *  at most count entries.
      */
-    void FindNextBlocksToDownload(NodeId nodeid, unsigned int count, std::vector<const CBlockIndex*>& vBlocks, NodeId& nodeStaller) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+    void FindNextBlocksToDownload(NodeId nodeid, const Peer& peer, unsigned int count, std::vector<const CBlockIndex*>& vBlocks, NodeId& nodeStaller) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Comment:
Perhaps it makes sense to remove `NodeId nodeid` as an argument to this function, now that it can easily be accessed from the `peer` argument.

Reply:
Yup, removed the NodeId arg.